src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.db;
+
+import java.util.HashMap;
+
+public class Storage {
+    public static final HashMap<String, Integer> FruitStorage = new HashMap<>();

```

Comment:
Unknown naming convention for the variable :)

```suggestion
    public static final HashMap<String, Integer> FRUIT_STORAGE = new HashMap<>();

```